src/utils/nftRewards.ts

```diff
@@ -388,12 +388,14 @@ export function sumTierFloors(
   rewardTiers: NftRewardTier[],
   tierIds?: number[],
 ) {
-  if (!tierIds) return 0
+  if (tierIds !== undefined && tierIds.length === 0) return 0
```

Comment:
Unrelated fix to this: 

<img width="626" alt="Screen Shot 2023-02-14 at 5 27 53 pm" src="https://user-images.githubusercontent.com/96150256/218668226-c62f1562-1505-4dac-aaab-fb0e30c5c161.png">